README.md

```diff
@@ -4,7 +4,52 @@ inspired by
 
 https://github.com/FredrikOseberg/react-chatbot-kit
 
----
+### Make your own configuration
+
+#### Questions`s roadmap
+
+First of all you need to make your own question`s roadmap dependencies.
+
+You need to create your `config.js` file.
+
+See example of `config.js` in directory `examples`.
+
+`config.js` file have four fields:
+
+1) initMessages - Array of messages that will see users when open a chat bot.
+2) initQuestions. It`s a start questions of roadmap for users. Array consist of questions names.
+3) Answers. Each answer have four fields:
+* Name. A custom name of answer, should be unique;
+* Content. Visible answer for users;
+* QuestionNames. Array of next questions, make your own dependencies;
+* Widgets (optionals field). Array of custom widgets shown in answer.
+4) Questions. Each question have three fields:
+* Name. A custom name of question, should be unique;
+* Text. Visible question for users;
+* AnswerName. A custom name of answer for this question.
+
+#### Add widgets
+
+You can add your own custom widgets for answers.
+
+Create your custom widget component. See examples in directory `examples/widgets`.
+
+Create object `widgets`, import and add all your widgets. 
+
+Create a function named `getWidget` which have param `name` and will return component from `widgets` object. Just like in file `examples/widgets/index.js`.
+
+### Now check how it`s worked
+
+Import function `createFlowBot` from package in your project.
+
+Import your config file and pass it as first parameter of function `createFlowBot`.
+
+`const flowBot = createFlowBot(config);`
+
+If you have a custom widgets, import your function `getWidget` and pass it in object `userDefinedOptions` - second parameter of function `createFlowBot`.
```

Comment:
If you want to use a custom widget, import your function `getWidget` and pass it into the second parameter of the function `createFlowBot`, `userDefinedOptions`.